app/models/building.ts

```diff
@@ -24,10 +24,10 @@ export default class Building extends BaseModel {
   @column()
   declare campusId: number
 
-  @column()
+  @column({ columnName: 'address_line1' })
```

Comment:
why did you do it?
not saying it is wrong, just asking to see idea behind it

Reply:
so basically lucid is changing model field names to camel case in default. So addressLine1 will transfer to address_line_1 and thus its no valid in current migrations implementation (in diagrams and migrations its address_line1) so i thought less invasive is to set it here for address_line1 but i can change it other way around

Reply:
makes sense, your version looks cleaner too
@Lennox2536 you might want to check this out, because you are using the same columns